e2e-test/cypress/integration/preselect_language_spec.js

```diff
@@ -0,0 +1,27 @@
+describe("Checking if language ist still selected after Logout", function () {
```

Comment:
Suggestion: "Check if language is still selected after logout"
rename the file to language_spec.js , describe "language", and specify test with "Check if language is still selected after logout"